staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -61,6 +70,18 @@ type CustomResourceDefinitionVersion struct {
 	// Storage flags the version as storage version. There must be exactly one flagged
 	// as storage version.
 	Storage bool
+	// Schema describes the schema for CustomResource used in validation, pruning, and defaulting.
+	// Setting this value overrides the global validation schema for this version.
+	// +optional
+	Schema *JSONSchemaProps
```

Comment:
adding schema here was not intended, was it?

Reply:
I added Schema API here because I wanted to keep the protobuf indexes consistent across PRs (schema = 4, subresources = 5, additionalPrinterColumns = 6).

Reply:
just leave out 4. I don't think any reordering takes place automatically.